File: single-language-analysis/dataset-files/java/TestSmell125.java
import org.junit.Test;
import static org.junit.Assert.*;

public class TestSmell125 {

    @Test
    public void testSuccess() {
        String outputMypy = OUTPUT_MYPY;
        for (String path : getTestCases(PASS_DIR)) {
            if (path.equals(outputMypy)) {
                String msg = "Unexpected mypy output\n\n";
                msg += String.join("\n", _stripFilename(outputMypy));
                throw new AssertionError(msg);
            }
        }
    }

    private String[] getTestCases(String passDir) {
        // Implementation of getTestCases
        return new String[0];
    }

    private String _stripFilename(String v) {
        // Implementation of _stripFilename
        return v;
    }
}

File: single-language-analysis/dataset-files/python/test_smell_125.py
# Original URL: https://github.com/ryfeus/lambda-packs/blob/74b21b9cddb57790c804ab973e26141eb7e5c912/ONNX-ARM/lambda-onnx-arm-3.8/numpy/typing/tests/test_typing.py#L99-L108

@pytest.mark.slow
@pytest.mark.skipif(NO_MYPY, reason="Mypy is not installed")
@pytest.mark.parametrize("path", get_test_cases(PASS_DIR))
def test_smell_125(path):
    # Alias `OUTPUT_MYPY` so that it appears in the local namespace
    output_mypy = OUTPUT_MYPY
    if path in output_mypy:
        msg = "Unexpected mypy output\n\n"
        msg += "\n".join(_strip_filename(v) for v in output_mypy[path])
        raise AssertionError(msg)